skpro/tests/test_polars.py

```diff
@@ -0,0 +1,121 @@
+"""Test file for polars dataframes"""
+
+import pandas as pd
+import polars as pl
+import pytest
+from sklearn.datasets import load_diabetes
+from sklearn.model_selection import train_test_split
+
+from skpro.datatypes import check_raise
+from skpro.datatypes._table._check import check_polars_table
+from skpro.datatypes._table._convert import convert_pandas_to_polars_eager
+from skpro.utils.validation._dependencies import _check_soft_dependencies
+
+X, y = load_diabetes(return_X_y=True, as_frame=True)
```

Comment:
data generation and loading should happen in a method or - optimally - pytest fixture, not on module load.
(we have too many of these in `sktime` and it slows test collection and imports)

Reply:
configured pytest fixture `polars_load_diabetes_pandas` to load polars X and y